libraries/Microsoft.Bot.Builder.LanguageGeneration/GetMethodExtensions.cs

```diff
@@ -34,11 +34,11 @@ public ExpressionEvaluator GetMethodX(string name)
             }
 
             // TODO: Should add verifiers and validators
-            switch (name)
-            {
-                case "join":
-                    return new ExpressionEvaluator("join", BuiltInFunctions.Apply(this.Join));
-            }
+            //switch (name)
```

Comment:
just remove those lines, use source control to remember history instead of comments